src/components/ViewCourse/sections/AddFastAddReserve.js

```diff
@@ -0,0 +1,72 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Button, Icon } from '@folio/stripes/components';
+import { CalloutContext, Pluggable, stripesConnect } from '@folio/stripes/core';
+
+/*
+  This component relies entirely on the create-inventory-records plugin existing and nothing
+  will be rendered by `Pluggable` if it doesn't. The unit tests for this app are written such that
+  the thing being tested is a stripes-cli-served bundle of _just_ this app - there are no plugins!
+  As a result, any of the code inside of Pluggable (onClose, renderTrigger, etc) can't be executed
+  during tests so we ignore it from code-coverage calculations. This is suboptimal, but barring
+  significant changes to the testing framework (just go on ahead and tell Mike Taylor to consider
```

Comment:
You can always go ahead and rewrite the tests yourself in your preferred framework. I believe that @JohnC-80 and @cowboyd have already begun that process.

Reply:
Oh hell no, I think the tests in this app are written wonderfully. If I rewrote it some other way then there would just be a different set of tradeoffs.  I just don't really love throwing down blanket `ignore` statements without spelling out the reasoning in code.

(All the snark in that comment is directed only at people who demand certain test coverage percentages, not devs :) )

Reply:
Haha, right! And you of course correct to add the justifying comment.